package.json

```diff
@@ -45,7 +45,7 @@
     "mocha": "^3.2.0",
     "mocha-jenkins-reporter": "^0.3.7",
     "simple-mock": "^0.7.3",
-    "voxa": "^2.2.0"
+    "voxa": "^2.2.1"
```

Comment:
i was just thinking we should start looking into peer dependencies, thoughts?

@rmberrios @wuelcas @wuelcas 
https://yarnpkg.com/lang/en/docs/dependency-types/#toc-peerdependencies

Reply:
I don't think `peerDependencies` would work in this specific project. I am using voxa 2.2.1 for this package because is public. For the simon skill (the project we're testing this package into, I'm using a private version of voxa. Maybe in the future ?